DataCatalog/examples/0.3-RELEASE/MobiDB_jsonld.json

```diff
@@ -0,0 +1,98 @@
+{
+  "@context": "http://schema.org/",
+  "@type": "DataCatalog",
+  "@id": "https://mobidb.org/",
+  "dct:conformsTo": "https://bioschemas.org/profiles/DataCatalog/0.3-RELEASE-2019_07_01",
```

Comment:
Please use the full URL here http://purl.org/dc/terms/conformsTo, I have seen that is the preferred way (based on other examples)